examples/stm32wl/src/bin/lorawan.rs

```diff
@@ -38,9 +38,22 @@ async fn main(_spawner: embassy::executor::Spawner, p: Peripherals) {
     static mut RADIO_STATE: SubGhzState<'static> = SubGhzState::new();
     let radio = unsafe { SubGhzRadio::new(&mut RADIO_STATE, radio, rfs, irq) };
 
-    let region = region::EU868::default().into();
+    let mut region: region::Configuration = region::EU868::default().into();
+
+    // NOTE: This is specific for TTN, as they have a special RX1 delay
```

Comment:
Shouldn’t this be automatically be updated by the CFList parser of lorawan crate already?